forward_command_controller/include/forward_command_controller/forward_joint_group_command_controller.h

```diff
@@ -98,6 +103,7 @@ class ForwardJointGroupCommandController: public controller_interface::Controlle
         return false;
       }
     }
+    commands_buffer_.readFromRT()->resize(n_joints_, 0.0);
```

Comment:
You can first `readFromNonRT()`, set the new values and then  `writeFronNonRT()`. You'll wait twice for the lock, once in each call, but that's still OK.

In fact, in the above usecase you don't actually need to read the buffer at all. You can simply prepare a vector and call `writeFromNonRT()`.